doc/tech/README.md

```diff
@@ -3,203 +3,170 @@
 
-Solving the problems associated with implementing the Abstraction Principle is a work in progress. CNTT has not solved all of the associated problems, nor has the industry. As technology evolves and more designs implement Cloud Native concepts, these problems will be addressed. This document is specifically intended as a one-stop-shop to capture CNTT policies which manage these situations as they exist today and their associated exceptions and transitions, and help drive the technology around and supporting the maturation of the Abstraction Principle.
+- **Mobile service provider 5G/4G virtual Radio Access Network (RAN) deployment and Edge Cloud B2B2X**: There are at least three use cases related to this (e.g. vRAN, VNF, Multi-access Edge Computing (MEC). While the use cases are different, the expectation is that they will run on the same infrastructure. So it makes sense to treat them together.
+  1. vRAN: Here the focus on virtual Baseband Unit (BBU) which has stringent requirements on processing for timing controls with 'remote radio heads'
+  1. NFV: Here the focus is on running the Core applications as virtual machines at the edge. This includes, vEPC elements, vRouters, Virtual Firewall (vFW), Virtual Load Balancer (vLB)
+  1. MEC: Here the focus is on running 3rd party or operator applications at the edge. The MEC resource pools could be supporting a variety of other MEC applications (smart city, v2x, consumer AR).
+- **Edge cloud service user**: For vRAN and NFV the user is the wireless network operator. For MEC it could be the operator, a 3rd party application provider or the wireless end-user.
+- **Edge cloud infrastructure user**: The Edge Cloud infrastructure user would be the network operator.
+- **Edge site(s)**: An operator's network could include thousands of sites. Each site could range from a handful of servers to dozens of racks.
+- **Connectviity reliability**: Front Haul reliability is driven by Radio requirements and is high. Backhaul reliability is driven by operator service requirements (5 9s)
+- **Edge Size**: medium to large
+- **Deployment infrastructure considerations / scaling**:
 
-Several specific technology areas have been identified by CNTT as using an ABI impacted by the Abstraction Principle, as follows:
-- SR-IOV
-- GPU/NPU
-- FPGA/Other Acceleration
-- CPU Instruction Sets and Extensions
+   - Case 2: By splitting CU-DU (from BBU), vCU can run on the same NFVI edge computing platform that runs UPF(5G) ( or S/P-GW-U in 4G) and other VNFs including vFW and vLB. End users/devices traffic can be released at the clear demarcation point placing UPF in Edge DC. We can install Kubernetes based cloud on top of NFVI and put UPF(and other VNFs) in front of Kubernetes to carry the user's traffic to cloud native apps in the Kubernetes.
```

Comment:
Given the discussion in #1168 can I suggest changing `NFVI edge computing` to `edge computing`?
Also suggest changing `We can install Kubernetes based cloud on top of NFVI and put UPF(and other VNFs) in front of Kubernetes to carry the user's traffic to cloud native apps in the Kubernetes.` to `We can adopt a hybrid model and put VM-based UPF (and other VNFs) to carry the user's traffic to CNFs running in Kubernetes running on the same cloud infrastructure.`

Reply:
fixed in: https://github.com/cntt-n/CNTT/pull/1267/commits/27bcef7a0f0ab40e3aa297bcf3c385a888997ee1